### PR TITLE
docs: Add clarification on event types in enhanced recording mode

### PR DESCRIPTION
**Pull request by webvictim**
The previous docs stated that the `enhanced_recording` field would always be set in audit log events generated when enhanced recording is enabled, which is false. This field only gets added to `session.end` events. Clarified this, as it confused a prospect.

Also added examples of what the `session.command` and `session.network` events look like.

### CODE REVIEW

**Review by stevenGravy — APPROVED** [review]

**Review by alexfornuto — APPROVED** [review]
👍 with one nit.

**Comment by alexfornuto** [comment]
@webvictim removed the auto-merge so you have the chance to parse my review comment.

File: docs/cspell.json [hunk 1]
```diff
@@ -763,6 +763,7 @@
     "webp",
     "webproxy",
     "webui",
+    "webvictim",
```
[the remark this answers is not in the document]
Comment (webvictim) [reply docs/cspell.json:766]: I saw we'd whitelisted Ben's name previously so figured it probably wasn't a big deal. I'm OK with changing all the usernames on the page to something else if it looks better, though.

[the remark this answers is not in the document]
Comment (alexfornuto) [reply docs/cspell.json:766]: If I'd seen that change I would have made the same comment there. Like I said, it's not a big enough deal as to block merging, I just wanted to make sure this was seen.

The more consistent we are with things like this the better for the reader, and the easier time we'll have when we eventually decide to go through and standardize them all.

**Pull request merged**